### PR TITLE
Add all open source strongloop repos

### PR DESCRIPTION
**Pull request by rmg**
Not all of StrongLoop's public repos are under open source licenses, so
they have to be listed individually :-(

@christo4ferris I finally got through the list.. It wasn't pretty.

### CODE REVIEW

File: orgs.js [hunk 1]
```diff
@@ -133,6 +133,264 @@
         "type": "org"},
         {"name": "strongloop",
```
Comment (christo4ferris) [inline orgs.js]: you need to remove the strongloop org if we are going to enumerate the repos or we'll be doing double accounting;-)

**Comment by christo4ferris** [comment]
@rmg thanks, see my comment.

File: orgs.js (hunk 1 above)
Comment (rmg) [inline orgs.js]: Weird, I thought 912db57 was on master. Fixed now :-)

**Comment by rmg** [comment]
@christo4ferris thanks. Comment addressed, so should be good to merge now 👍

**Pull request merged**